### DISCUSSION TITLE
Hide SteamDeck search bar

### DISCUSSION THREAD

**Discussion by ZongerX** [jessebofill/DeckWebBrowser#20] [Ideas]
I use web browser and SteamDeck with TV, and sometimes use it to watch youtube, but DeckWebBrowser doesn't hide the top panel and watching youtube is uncomfortable. 

Also when I use DeckWebBrowser on SteamDeck OLED then I kill OLED module because top symbols is burning OLED screen. If static image stay too much time it's too bad for screen. 

Can you fix it with adding option for hide search bar?

**Comment by jessebofill** [comment]
Hello, thanks for your feedback. Currently the plugin already allows to hide the tab bar (in the menu by pressing start button). Unfortunately, modifying the search bar is not so simple as steam automatically renders it, and manipulating it is tricky and causes unexpected behavior.
